webpack/scenes/SmartProxy/AdditionalCapsuleContent.js

```diff
@@ -55,6 +56,11 @@ const AdditionalCapsuleContent = ({ counts }) => {
         {`${dockerManifestCount} Container manifests`}<br />
       </>
             }
+      {dockerManifestListCount > 0 &&
+      <>
+        {`${dockerManifestListCount} Container manifest lists`}<br />
```

Comment:
One thing I realized -- it doesn't look like the container bits are getting translated (except ansible collections?) for some reason:

![image](https://github.com/Katello/katello/assets/14796566/87ae39e3-a8af-4612-9389-256c53862fb8)

vs

![image](https://github.com/Katello/katello/assets/14796566/2e8ff03a-8c17-40d6-be2a-a2e17611ff89)